fhir-server-webapp/src/main/liberty/config/configDropins/disabled/jvm.options

```diff
@@ -1,8 +1,9 @@
-# Set a fixed max heap size
--Xmx3072M
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2021
+# 
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
 
-# Turn on verbose classloading
-#-verbose:class
-
-# Turn on verbose garbage collection
-#-verbose:gc
+# https://www.eclipse.org/openj9/docs/xxinitialrampercentage/
+-XX:InitialRAMPercentage=50.00
+-XX:MaxRAMPercentage=90.00
```

Comment:
```
<gc-end id="443" type="scavenge" contextid="439" durationms="57.408" usertimems="223.188" systemtimems="0.172" stalltimems="4.439" timestamp="2021-12-20T18:00:43.360" activeThreads="4">

  <mem-info id="444" free="469525144" total="858914816" percent="54">

    <mem type="nursery" free="269655128" total="468451328" percent="57">

      <mem type="allocate" free="269655128" total="349700096" percent="77" />

      <mem type="survivor" free="0" total="118751232" percent="0" />

    </mem>

    <mem type="tenure" free="199870016" total="390463488" percent="51" macro-fragmented="99381968">

      <mem type="soa" free="180346432" total="370939904" percent="48" />

      <mem type="loa" free="19523584" total="19523584" percent="100" />

    </mem>

    <pending-finalizers system="0" default="1887" reference="1537" classloader="0" />

    <remembered-set count="18969" />

  </mem-info>

</gc-end>

<cycle-end id="445" type="scavenge" contextid="439" timestamp="2021-12-20T18:00:43.361" />

<allocation-satisfied id="446" threadId="000000000070AB00" bytesRequested="48" />

<af-end id="447" timestamp="2021-12-20T18:00:43.361" threadId="000000000070B508" success="true" from="nursery"/>

<exclusive-end id="448" timestamp="2021-12-20T18:00:43.361" durationms="58.933" />
```
I analyzed the settings, and it is able to grow from the default 469M to the maximum supported in a 2GB container.

Reply:
given that the openj9 defaults for max heap are never more than 75%, I'm a tad nervous that 90% is too aggressive

Reply:
What else is running in the container? I can only think of curl, curl is really designed to be memory efficient. For 2G allocation, that's 200M. 4G is 400M.

Reply:
I didn't account for the tty, so the shell is one thing additional.

Reply:
I think its more the 'off-heap' storage that i'd be worried about.  I'm not sure whether the zip stream stuff uses much memory or not.  also not sure how big the stacks can be.  90% may be totally fine...was just communicating a gut feel

Reply:
What about all the class/JIT and stack space? 80% memory utilization is a common threshold for alerting, so I agree that 90% is probably too aggressive.